src/ObjectUtils/CoffFile.cpp

```diff
@@ -120,6 +122,31 @@ uint64_t CoffFileImpl::GetExecutableSegmentOffset() const {
 bool CoffFileImpl::IsElf() const { return false; }
 bool CoffFileImpl::IsCoff() const { return true; }
 
+std::optional<PdbDebugInfo> CoffFileImpl::GetDebugPdbInfo() const {
```

Comment:
Since getDebugPDBInfo can return a error consider making this ErrorMessageOr?

Reply:
Done.